ci/terraform/canary-sign-in-with-ipv.tf

```diff
@@ -32,8 +32,7 @@ module "canary_sign_in_with_ipv" {
   client_private_key  = tls_private_key.stub_rp_client_private_key[0].private_key_pem
   issuer_base_url     = var.use_integration_env_for_sign_in_journey ? var.integration_issuer_base_url : var.issuer_base_url
 
-  # the test will run Mon-Fri, between 0800-1700 (UTC) every 5 minutes
-  smoke_test_cron_expression = "0/05 08-17 ? * MON-FRI *"
+  smoke_test_cron_expression = var.smoke_test_cron_expression
```

Comment:
is the frequency defined somewhere as that's in the cron expression?

Reply:
In the override files